tensorflow/core/lib/wav/wav_io.cc

```diff
@@ -223,13 +223,13 @@ Status DecodeLin16WaveAsFloatVector(const string& wav_string,
       ReadValue<uint32>(wav_string, &format_chunk_size, &offset));
   if ((format_chunk_size != 16) && (format_chunk_size != 18)) {
     return errors::InvalidArgument(
-        "Bad file size for WAV: Expected 16 or 18, but got", format_chunk_size);
+        "Bad file size for WAV: Expected 16 or 18, but got ", format_chunk_size);
```

Comment:
chunk size instead of file size?

Reply:
Not sure about that piece of code. 

Was focused on the "audio format number" error that did not make sense to me.